src/components/Artists/Artists.scss

```diff
@@ -1,6 +1,14 @@
 .artists {
   width: 100%;
 
+  &__progressbar {
```

Comment:
why do you need these progress bar styles?

Reply:
Old code. Deleted